charts/argocd-image-updater/templates/_helpers.tpl

```diff
@@ -38,7 +38,7 @@ Common labels
 helm.sh/chart: {{ include "argocd-image-updater.chart" . }}
 {{ include "argocd-image-updater.selectorLabels" . }}
 {{- if .Chart.AppVersion }}
-app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | trunc 63 | quote }}
```

Comment:
for some reason this chart doesn't allow `image.tag` overrides. if it did, I have to admit, this would be more useful.

example where this was useful: #1693

if you don't want this feel free to close the PR.